products/antixlinux.md

```diff
@@ -0,0 +1,60 @@
+---
+permalink: /antix
+alternate_urls:
+-   /antixlinux
+-   /antix-linux
+title: antiX
+category: os
+releasePolicyLink: https://www.antixforum.com/forums/topic/when-is-end-of-support-for-stable-antix-versions-17-19/#post-26424
+iconSlug: mxlinux
```

Comment:
```suggestion
iconSlug: NA
```

The antix website doesn't re-use the mx logo, so we shouldn't either.

Reply:
@alexkoster why did you mark this resolved ? did i miss anything about this one

Reply:
> @alexkoster why did you mark this resolved ? did i miss anything about this one
@usta My apologies - I think I pushed the wrong button on the mobile app but I went ahead and removed the iconSlug line in the latest update to the [antiX branch](https://github.com/alexkoster/endoflife.date/tree/antix).  Or feel free to let me know or make modifications to keep the iconSlug line but with a value of NA. 

I was working on a commit to the [simple-icons](https://github.com/simple-icons/simple-icons) repository for antiX and Devuan but was having some trouble with scaling and centering the SVG properly.  I hope to request help and make those commits in the future so antiX and Devuan will feature icons.

Thanks again for your time and please let me know if you have any further suggestions for changes or updates.

Reply:
Adding `iconSlug: NA` back. We can have a separate PR to add the icon again once the simple-icons repo has them